src/main/java/org/sonarsource/auth/github/GitHubIdentityProvider.java

```diff
@@ -144,6 +145,14 @@ private String getLogin(GsonUser gsonUser) {
     }
   }
 
+  private static String getName(GsonUser gsonUser){
+    String name = gsonUser.getName();
+    if (isNullOrEmpty(name)) {
```

Comment:
you can inline with
`return isNullOrEmpty(name) ?
  gsonUser.getLogin()
  : name;`